test/avocado/timeoutlib.py

```diff
@@ -141,7 +144,7 @@ class IFailedError(Exception):
     white_horse = []
 
     # Simple "try so many times, and die" case
-    @Retry(attempts = 5, error = IFailedError('Too many retries!'))
+    @Retry(attempts = 5, exceptions=(IFailedError, ), error = IFailedError('Too many retries!'))
```

Comment:
You're inconsistent between (a, ) and (a,) in this patch.  PEP8 seems to prefer without the space.

Should you even use a tuple at all here?  I don't see any reason why you couldn't just directly give the exception, in case you only want to catch a single one.

Reply:
I thought I always used the form with trailing space; you mean inconsistent with the rest of the code? I'm happy to change it. `Retry` expects a tuple.